operations/CSM_product_management/Configure_Non_Compute_Nodes.md

```diff
@@ -0,0 +1,168 @@
+# Configure Non-compute Nodes with CFS
+
+Non-compute node (NCN) personalization applies post-boot configuration to the
+HPE Cray EX management nodes. Several HPE Cray EX produce environments outside
+of CSM require NCN personalization to function. Consult the manual for each
+product to configure them on NCNs.
```

Comment:
It might be good to reference the new Getting Started guide here, as that guides the admin through the install/upgrade processes and will call out which products need to perform customization at each step in the install/upgrade workflow.  

You can find the Getting Started guide at https://connect.us.cray.com/confluence/pages/viewpage.action?spaceKey=TechPubs&title=EX+System+%28V1.5%29+Software+Documentation (it is still under construction)

FWIW, COS refers to it like this (just one example): "This procedure applies to COS upgrades where an existing configuration exists. The existing configuration will likely include other Cray EX product entries. Update the COS entry as described in this section. The HPE Cray EX System Software Getting Started Guide provides guidance on how and when to update the entries for the other products."

Reply:
Fixed.